rulesets/resources/ES_ZERHruleset.rb

```diff
@@ -1048,7 +1048,11 @@ def self.set_appliances_refrigerator_reference(orig_bldg, new_bldg)
       location = refrigerator.location.gsub('unvented', 'vented')
     end
 
-    rated_annual_kwh = lookup_reference_value('refrigerator_rated_annual_kwh')
+    if @program_version == ESConstants::SFNationalVer3_3
+      rated_annual_kwh = Defaults.get_refrigerator_values(@nbeds)[:rated_annual_kwh]
+    else
+      rated_annual_kwh = lookup_reference_value('refrigerator_rated_annual_kwh')
+    end
```

Comment:
It'd be nice not to introduce version-specific logic in here, we'd like as much logic as possible encapsulated in the TSV files. Can you just check `if rated_annual_kwh.nil?`?